RabbitMQ.Stream.Client/RoutingClient.cs

```diff
@@ -180,8 +186,14 @@ await LookupConnection(clientParameters, metaDataInfo.Leader, MaxAttempts(metaDa
         public static async Task<IClient> LookupRandomConnection(ClientParameters clientParameters,
```

Comment:
@Gsantomaggio I would also correct the comment specifying the new behaviour

Reply:
done via https://github.com/rabbitmq/rabbitmq-stream-dotnet-client/pull/352/commits/cfb6b241d4007a65d9ebf84d717f6afddfdd5b76